.github/workflows/pretest-rocm-test.sh

```diff
@@ -2,7 +2,7 @@
 
 set -uex
 
-# Python 3.8 (Ubuntu 20.04)
```

Comment:
ah sorry I overlooked this, this scirpt is inside the `rocm/dev-ubuntu-20.04:5.0-complete` docker container